docs/pallas/tpu/pipelining.md

```diff
@@ -11,22 +11,33 @@ kernelspec:
   name: python3
 ---
 
++++ {"id": "7704d3bb"}
+
 (pallas_tpu_pipelining)=
 
 +++ {"id": "teoJ_fUwlu0l"}
 
-# Pipelining
+# TPU Pipelining
 
 <!--* freshness: { reviewed: '2024-04-08' } *-->
 
 +++ {"id": "gAJDZh1gBh-h"}
 
-In this guide we'll cover how memory spaces in TPU work and how to write
-pipelines in Pallas that overlap memory I/O with compute.
+This guide serves as a reference for TPU-specific pipelining concerns.
+We'll review the memory hierarchy and compute units on TPUs, and TPU-specific features of the pipelining API. For a more general-purpose overview of pipelining, see the [software pipelining guide](https://docs.jax.dev/en/latest/pallas/pipelining.html).
 
 ```{code-cell}
-:id: ejAVO6ikUUuF
-
+---
+executionInfo:
+  elapsed: 54
+  status: ok
+  timestamp: 1744908474512
+  user:
+    displayName: Justin Fu
+    userId: '17543197034567316452'
+  user_tz: 420
+id: ejAVO6ikUUuF
+---
```

Comment:
Should this be here?

Reply:
It's auto-generated from Jupytext sync, so I'd rather not touch it in case it breaks something.